### PR TITLE
asyncio + aiohttp implementation

### PR DESCRIPTION
**Pull request by gdraynz**

### CODE REVIEW

**Comment by gdraynz** [comment]
Woops, indeed :bow:

**Comment by guilhem** [comment]
Maybe using https://pypi.python.org/pypi/setuptools_scm for version can be better idea
You may have a look

**Comment by guilhem** [comment]
And be careful to end-of-file ;)

File: VERSION.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+0.1.1.dev0
```
Comment (lerignoux) [inline VERSION.txt]: shouldn't it switch to 0.1.2 ?

**Comment by lerignoux** [comment]
Better add a more exhaustive README.

mainly how to use it.

File: VERSION.txt (hunk 1 above)
Comment (gdraynz) [inline VERSION.txt]: Let's just bump versions within master

**Pull request merged**